samples/monster_generated.h

```diff
@@ -814,6 +818,102 @@ inline flatbuffers::unique_ptr<MonsterT> UnPackMonster(
   return flatbuffers::unique_ptr<MonsterT>(GetMonster(buf)->UnPack(res));
 }
 
+inline const uint8_t *GetMonsterBinarySchema(bool endPtr= false) {
+    // Buffer containing the binary schema.
+    static const uint8_t bfbsData[] = {
+        0x18,0x00,0x00,0x00,0x42,0x46,0x42,0x53,0x10,0x00,0x1C,0x00,0x04,0x00,0x08,0x00,0x0C,0x00,0x10,0x00,
```

Comment:
I understand we'd actually want to test this functionality, but this is going this is going to generate a lot of churn whenever this binary changes. Since it is somewhat niche functionality, I'd almost prefer it if by default we do not turn this on for the code generated in this repo.

Also, I'd think it be even better if it sat in its own header (since then it is easy to not pay the overhead for all the users of this header that don't use this function call). The way it is now, this will generate a lot of data in each compilation unit it is included in, some which will be optimized by the linker, some will not. When I said that it is ok to put this data in a header, I meant for it to be the ONLY thing in the header.. not even with a inline function around it (it be good if you got an error if you included this twice :)